health-services/project-factory/src/server/validators/campaignValidators.ts

```
@@ -576,21 +650,42 @@
+        Object.fromEntries(
+            Object.entries(row).map(([key, value]) =>
+                [key, typeof value === "string" ? value.trim() : value]
+            )
+        )
+    );
+    const seen = new Set<string>();
+    const duplicateRowNumbers: string[] = [];
+    for (const row of boundaryData) {
+        const rowNumber = row["!row#number!"];
+        const rowCopy = { ...row };
+        delete rowCopy["!row#number!"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)